integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -454,9 +454,9 @@ public void testHourOfDayWithUnderscores() throws IOException {
         "hour_of_day(timestamp('2020-09-16 17:30:00'))", null, "integer"));
     verifyDataRows(result, rows(17));
 
-    result = executeQuery("select hour_of_day(datetime('2020-09-16 17:30:00'))");
+    result = executeQuery("select hour_of_day(timestamp('2020-09-16 17:30:00'))");
```

Comment:
This part is a duplicate of the above part now.

Reply:
removed in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/1aec4f9ee0ca2e447f3a4b9cdff2169a98c76fcb